app/controllers/concerns/form_attachment_create.rb

```diff
@@ -5,11 +5,28 @@ module FormAttachmentCreate
   include SentryLogging
 
   def create
+    debug_timestamp = Time.current.iso8601
+    if Flipper.enabled?(:hca_log_form_attachment_create)
+      log_message_to_sentry(
```

Comment:
Any concerns (hah, it's in a concern) that this concern is used by a few other non 1010 controllers? It would just an extra log so probably not a big deal but I wanted to call it out.

Reply:
Yes, which is part of why we want to be extra-cautious by feature-flagging it. If we see a weird change in app behavior, off it goes.